README.md

```diff
@@ -2,6 +2,9 @@
 
 `oc` client in a container image.
 
+[![dockeri.co](https://dockeri.co/image/appuio/oc)](https://hub.docker.com/r/appuio/oc/)
```

Comment:
I feel it looks better if this badge is totally on top of the document, i.e. the first thing after the title. WDYT?

Reply:
While you're at it ... why not explain what "oc" is? Mentioning "OpenShift" shouldn't harm (but could help to understand what this is about).

Reply:
I didn't like the huge image on top but i found a more compact badge to use.
Added a few words about what this project is for